### PR TITLE
Add "epoch window" for handle-based freshness

### LINKED ISSUE #283: Discuss "epoch window" for handles
The handle-related portion of the Freshness section should provide some guidance on the extent of the "epoch window", i.e., the number of previously received handles that need to be kept by an appraiser to deal with propagation delay.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #283

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -1156,6 +1156,17 @@ in case of mismatching handles, or by buffering incoming messages
 that might be associated with a handle that the receiver has not yet
 obtained.
 
+More generally, in order to prevent an appraising entity from generating false
+negatives (e.g., discarding Evidence that it is deemed stale even if it is
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1160]:
```suggestion
negatives (e.g., discarding Evidence that is deemed stale even if it is
```

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -1156,6 +1156,17 @@ in case of mismatching handles, or by buffering incoming messages
...
+not), the appraising entity should keep an "epoch window" consisting of the
+most recently received handles.  The depth of such epoch window is directly
+proportional to the maximum network propagation delay and inversely
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1163]: the network prop delay point is not clear.  why?

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -1156,6 +1156,17 @@ in case of mismatching handles, or by buffering incoming messages
...
+proportional to the epoch duration.  The appraising entity shall compare the
+handle carried in the received Evidence or Attestation Result with the handles
+in its epoch window to find a suitable match.  Note that if the network
+round-trip time is negligible, the depth of the epoch window might be 1, i.e.,
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1167]: There's still race conditions, so is anything negligible?

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -1156,6 +1156,17 @@ in case of mismatching handles, or by buffering incoming messages
 that might be associated with a handle that the receiver has not yet
 obtained.
 
+More generally, in order to prevent an appraising entity from generating false
+negatives (e.g., discarding Evidence that is deemed stale even if it is
+not), the appraising entity should keep an "epoch window" consisting of the
+most recently received handles.  The depth of such epoch window is directly
+proportional to the maximum network propagation delay and inversely
+proportional to the epoch duration.  The appraising entity shall compare the
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1164]:
```suggestion
proportional to the maximum network propagation delay between the first to receive the handle and the last to receive the handle, and it is inversely proportional to the epoch duration.  
The appraising entity shall compare the
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -1156,6 +1156,17 @@ in case of mismatching handles, or by buffering incoming messages
...
+handle carried in the received Evidence or Attestation Result with the handles
+in its epoch window to find a suitable match.  Note that if the network
+round-trip time is negligible, the depth of the epoch window might be 1, i.e.,
+consisting only of the handle associated with the current epoch.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1168]:
```suggestion
in its epoch window to find a suitable match.  
```

**Pull request merged**